crates/typst/src/geom/ratio.rs

```diff
@@ -134,6 +134,12 @@ impl Div<Ratio> for f64 {
     }
 }
 
+impl From<Abs> for Ratio {
+    fn from(value: Abs) -> Self {
+        Self::new(value.to_pt())
+    }
+}
```

Comment:
This impl feels _very_ wrong. What is it used for?

Reply:
I mean I don't see what's so wrong about it, but it *was* used to make some `Transform::scale` a bit less of a chore to write, but it's okay it's removed

Reply:
It's wrong in my opinion because there is no natural conversion from a length to a ratio. Why should 50pt coerce into 50%? It doesn't make sense in isolation.